src/sql/azdata.d.ts

```diff
@@ -942,6 +942,7 @@ declare module 'azdata' {
 		rowCount: number;
 		columnInfo: IDbColumn[];
 		complete: boolean;
+		showplangraph?: any;
```

Comment:
`showPlanGraph`

(but this should also go into proposed file. And please add comments for all new API additions)

Reply:
This should also ideally have a concrete type associated with it. Is there a reason you needed to make this `any`?